client/packages/invoices/src/InboundShipment/DetailView/SidePanel/SidePanel.tsx

```diff
@@ -13,20 +16,46 @@ import { RelatedDocumentsSection } from './RelatedDocumentsSection';
 import { TransportSection } from './TransportSection';
 
 export const SidePanel: FC = () => {
-  const { data } = useInbound.document.get();
+  const t = useTranslation('replenishment');
   const { success } = useNotification();
-  const t = useTranslation('common');
+  const { data } = useInbound.document.get();
+  const { mutateAsync } = useInbound.document.delete();
   const isTransfer = !!data?.linkedShipment?.id;
+  const canDelete = data ? data?.status === InvoiceNodeStatus.New : false;
```

Comment:
equivalent to this, if data is undefined then `data?.status` is undefined and not equal to `New`
```suggestion
  const canDelete = data?.status === InvoiceNodeStatus.New;
```

Reply:
[e2277b9](https://github.com/openmsupply/open-msupply/pull/2218/commits/e2277b94130e87e7e682fde0fcffbbee2097a4cf)